tokamak.py

```diff
@@ -3,6 +3,7 @@
 import yaml
 import numpy as np
 
+
 def parse_args():
```

Comment:
I'd suggest renaming this to something slightly different, as it has the same name as `parse_args` method of `argparse`, as in line 14. (It probably won't make any problem here as namespace is different, but just for good? practice.)